src/slice.cpp

```diff
@@ -66,6 +93,12 @@ void Slice::start_next_process(time_point current_time)
         return start_next_process(current_time);
     }
 
+    using namespace std::chrono;
+    schedule_logger.log(fmt::format("{:6}: cpus={} cmd='{}' budget={}\n",
+                                    duration_cast<milliseconds>(current_time - start_time).count(),
+                                    cpus.as_list(),
+                                    running_process->argv,
```

Comment:
Maybe it would be more readable to print the process partition and index (something like `BE1/proc0`), instead of full command (which might be quite long).

Reply:
Yes, but for test cases, you can now use the `dummy` command, where you can give individual processes nice "names" as parameters. This makes it easier to figure out how does the schedule correspond to the configuration. You can use anonymous partitions and don't have to think about the names. If you use anonymous partitions with many processes, it might be quite challenging to figure out which process in the schedule corresponds to which process in the configuration.